src/test/ui/feature-gates/feature-gate-cfg-version.rs

```diff
@@ -1,28 +1,24 @@
-#[cfg(version("1.44"))]
+#[cfg(version(1.45))]
 //~^ ERROR `cfg(version)` is experimental and subject to change
 fn foo() -> bool { true }
-#[cfg(not(version("1.44")))]
+#[cfg(not(version(1.45)))]
 //~^ ERROR `cfg(version)` is experimental and subject to change
 fn foo() -> bool { false }
 
-#[cfg(version("1.43", "1.44", "1.45"))] //~ ERROR: expected single string literal
+#[cfg(version(99.99.99))]
```

Comment:
Is there a test for an overflowing version? Such as `99999`?

Reply:
Or how about negative version?

Reply:
Good idea, added overflow and underflow tests